x-pack/test/apm_api_integration/tests/settings/agent_configuration/agent_configuration.spec.ts

```diff
@@ -383,7 +383,8 @@ export default function agentConfigurationTests({ getService }: FtrProviderConte
           expect(await waitFor(hasBeenAppliedByAgent)).to.be(true);
         });
       });
-    }
+    },
+    true
```

Comment:
Can you use `registry.when.skip` instead? This is near impossible to spot.

Reply:
I couldn't, the skip is a value, in when but I will take care soon, and fix it